cicd/builds/custom-builds-buildah.adoc

```diff
@@ -11,7 +11,7 @@ nodes. This means the _mount docker socket_ option of a custom build is not
 guaranteed to provide an accessible docker socket for use within a custom build
 image.
 
-If you require this capability in order to build and push images, add the Buildah
+If you require this capability to build and push images, add the Buildah
```

Comment:
Uncertain if the sentence means the same with the change.

Reply:
I agree, will revert this one.